spec/rails_helper.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../config/environment', __dir__)
+abort('The Rails environment is running in production mode!') if Rails.env.production?
```

Comment:
What the purpose of the line?

Reply:
Prevent database truncation if the environment is production

Reply:
Will you run tests on production?

Reply:
No